### PR TITLE
mongod: add page

### LINKED ISSUE #1176: page request: mongod
This is the command that runs the server

https://linux.die.net/man/1/mongod

### PR DESCRIPTION
**Pull request by YangVincent**
Fixes #1176 

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by YangVincent** [comment]
#1176

File: pages/common/mongod.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# mongod
+
+> The daemon process for MongoDB.
```
Comment (agnivade) [inline pages/common/mongod.md]: I suggest changing this line to "The MongoDB database server"

File: pages/common/mongod.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Specify runtime configurations:
```
Comment (agnivade) [inline pages/common/mongod.md]: runtime configurations => a config file

File: pages/common/mongod.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`mongod --config <filename>`
```
Comment (agnivade) [inline pages/common/mongod.md]: `<filename> => {{filename}}`

File: pages/common/mongod.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- Specify port for MongoDB to listen to client connections:
```
Comment (agnivade) [inline pages/common/mongod.md]: Specify the port to listen on.

File: pages/common/mongod.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+`mongod --port <port>`
```
Comment (agnivade) [inline pages/common/mongod.md]: `<port> => {{port}}`

File: pages/common/mongod.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+- Change database profiling level. 0 is off, 1 is only slow operations, 2 is all:
```
Comment (agnivade) [inline pages/common/mongod.md]: Change => Specify. And wrap the last sentence in parenthesis.

File: pages/common/mongod.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+`mongod --profile <level>`
```
Comment (agnivade) [inline pages/common/mongod.md]: <level> => {{0|1|2}}

**Comment by YangVincent** [comment]
Sure, I'd love that! I use tldr pretty often so it's nice to be able to contribute to the project

**Comment by agnivade** [comment]
I think we should let @YangVincent hang around for a little more time to get a feel for the overall theme for this project, before granting him higher privileges.

**Comment by sbrl** [comment]
Good idea, @agnivade

**Pull request merged**